ros1_foxglove_bridge/src/ros1_foxglove_bridge_nodelet.cpp

```diff
@@ -59,7 +60,7 @@ class FoxgloveBridge : public nodelet::Nodelet {
                                                std::placeholders::_1, std::placeholders::_2));
       _server->start(address, static_cast<uint16_t>(port));
 
-      _updateTimer = getMTNodeHandle().createTimer(ros::Duration(max_update_ms / 1e3),
+      _updateTimer = getMTNodeHandle().createTimer(ros::Duration(MIN_UPDATE_PERIOD_MS / 1e3),
```

Comment:
Why create a timer here at all? I think you can just directly call `updateAdvertisedTopics()`